inputfiles/overridingTypes.json

```diff
@@ -2400,6 +2455,253 @@
+                        "read": {
+                            "override-signatures": [
+                                "read<T extends ArrayBufferView>(view: T): Promise<ReadableStreamReadResult<T>>"
+                            ]
+                        }
+                    }
+                }
+            },
+            "ReadableStreamDefaultReader": {
+                "type-parameters": [
+                    {
+                        "name": "R",
+                        "default": "any"
+                    }
+                ],
+                "constructor": {
+                    "override-signatures": [
+                        "new<R = any>(stream: ReadableStream<R>): ReadableStreamDefaultReader<R>"
+                    ]
+                },
+                "methods": {
+                    "method": {
+                        "read": {
+                            "override-signatures": [
+                                "read(): Promise<ReadableStreamReadResult<R>>"
+                            ]
+                        }
+                    }
+                }
+            },
+            "ReadableByteStreamController": {
+                "properties": {
+                    "property": {
+                        "byobRequest": {
+                            "override-type": "ReadableStreamBYOBRequest | undefined"
+                        }
+                    }
+                }
+            },
+            "ReadableStreamDefaultController": {
+                "type-parameters": [
+                    {
+                        "name": "R",
+                        "default": "any"
+                    }
+                ],
+                "methods": {
+                    "method": {
+                        "enqueue": {
+                            "override-signatures": [
+                                "enqueue(chunk: R): void"
+                            ]
+                        }
+                    }
+                }
+            },
+            "ReadableWritableStreamPair": {
+                "name": "ReadableWritableStreamPair",
+                "type-parameters": [
+                    {
+                        "name": "R",
+                        "extends": "ReadableStream"
+                    },
+                    {
+                        "name": "W",
+                        "extends": "WritableStream"
+                    }
+                ],
+                "override-exposed": "Window Worker",
+                "properties": {
+                    "property": {
+                        "readable": {
+                            "name": "readable",
+                            "override-type": "R"
+                        },
+                        "writable": {
+                            "name": "writable",
+                            "override-type": "W"
+                        }
+                    }
+                },
+                "no-interface-object": "1"
```

Comment:
1. A dictionary should be in dictionary part of overridingTypes.json.
2. A dictionary doesn't need `override-exposed`.

Reply:
This is probably a leftover from my initial hack to make `<R extends ReadableStream>` work. I'll look into it.